modules/getintentBidAdapter.js

```diff
@@ -1,78 +1,130 @@
-import { STATUS } from 'src/constants';
-import adaptermanager from 'src/adaptermanager';
+import { registerBidder } from 'src/adapters/bidderFactory';
 
-var bidfactory = require('src/bidfactory.js');
-var bidmanager = require('src/bidmanager.js');
-var adloader = require('src/adloader.js');
+const BIDDER_CODE = 'getintent';
+const BID_HOST = 'px.adhigh.net';
+const BID_BANNER_PATH = '/rtb/direct_banner';
+const BID_VIDEO_PATH = '/rtb/direct_vast';
+const VIDEO_PROPERTIES = [
+  'protocols', 'mimes', 'min_dur', 'max_dur', 'min_btr', 'max_btr', 'vi_format', 'api', 'skippable'
+];
+const OPTIONAL_PROPERTIES = [
+  'cur', 'floor'
+];
 
-var GetIntentAdapter = function GetIntentAdapter() {
-  var headerBiddingStaticJS = window.location.protocol + '//cdn.adhigh.net/adserver/hb.js';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['getintentAdapter'],
+  supportedMediaTypes: ['video', 'banner'],
 
-  function _callBids(params) {
-    if (typeof window.gi_hb === 'undefined') {
-      adloader.loadScript(headerBiddingStaticJS, function() {
-        bid(params);
-      }, true);
-    } else {
-      bid(params);
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid to validate.
+   * @return {boolean} True if this is a valid bid, and false otherwise.
+   * */
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.pid && bid.params.tid);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} bidRequests - an array of bids.
+   * @return ServerRequest[]
+   */
+  buildRequests: function(bidRequests) {
+    return bidRequests.map(bidRequest => {
+      let giBidRequest = buildGiBidRequest(bidRequest);
+      return {
+        method: 'GET',
+        url: buildUrl(giBidRequest),
+        data: giBidRequest,
+      };
+    });
+  },
+
+  /**
+   * Callback for bids, after the call to DSP completes.
+   * Parse the response from the server into a list of bids.
+   *
+   * @param {object} serverResponse A response from the server.
+   * @param {BidRequest} bidRequest
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse, bidRequest) {
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so doing something like

```JavaScript
serverResponse = serverResponse.body
```

(or however you'd prefer) in this function should give you the expected behavior